types/braintree/index.d.ts

```diff
@@ -16,11 +20,11 @@ declare namespace braintree {
         Development = 'Development',
         Production = 'Production',
         Qa = 'Qa',
-        Sandbox = 'Sandbox'
+        Sandbox = 'Sandbox',
     }
 
     export interface GatewayConfig {
-        environment: any;
+        environment: Environment;
```

Comment:
👍

Reply:
Thanks a lot for reviewing and approving this PR